src/main/kotlin/it/pagopa/wallet/eventdispatcher/service/WalletCDCService.kt

```diff
@@ -19,17 +18,27 @@ class WalletCDCService(
 
     private val log = LoggerFactory.getLogger(WalletCDCService::class.java.name)
 
-    fun sendToKafka(event: LoggingEvent): Mono<Unit> {
-        log.info("Sending CDC event to Kafka: [{}]", event.id)
-
+    fun sendToKafka(event: WalletLoggingEvent): Mono<Unit> {
         return Mono.defer {
                 cdcKafkaTemplate
-                    .send(cdcTopicName, event.id, event)
+                    .send(cdcTopicName, event.walletId, event)
                     .doOnSuccess {
-                        log.info("Successfully sent CDC event to Kafka: [{}]", event.id)
+                        log.info(
+                            "Succesfully sent event with id [{}] of type [{}] with walletId [{}] published on [{}]",
+                            event.id,
+                            event.type,
+                            event.walletId,
+                            event.timestamp
+                        )
                     }
                     .doOnError {
-                        log.error("Failed to send CDC event to Kafka: [{}]", event.id, it)
+                        log.error(
```

Comment:
We should also log the error to have an understanding of what has happened

Reply:
done here 57fe26c666da766168a9b9a4c3fec611a602600b